dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/ShortestResponseLoadBalance.java

```diff
@@ -36,6 +42,46 @@
 
     public static final String NAME = "shortestresponse";
 
+    private static final int SLIDE_PERIOD = Integer.parseInt(System.getProperty("ShortestResponseLoadBalance.slidePeriod", "30000"));
```

Comment:
change to use `ApplicationModel.getEnvironment().getConfiguration().getProperty`